src/hearings/containers/request-hearing/hearing-edit-summary/judge-details-section/judge-details-section.component.html

```diff
@@ -19,7 +19,7 @@ <h2 class="govuk-heading-m">Judge details
         (click)="onChange('needJudge')">Change</a>
     </div>
   </div>
-  <div class="govuk-summary-list__row" *ngIf="judgeName?.length > 0">
```

Comment:
why this condition is removed?
Please check all the other conditions removed in this file as well